package.json

```diff
@@ -2,6 +2,7 @@
   "name": "examples",
   "license": "MIT",
   "private": true,
+  "packageManager": "pnpm@8.14.1+sha256.2df78e65d433d7693b9d3fbdaf431b2d96bb4f96a2ffecd51a50efe16e50a6a8",
```

Comment:
You may need to specify engines.pnpm to do
 double-check in case someone forget enable corepack

Reply:
@hardfist Good point! Added 👍

Reply:
Nice catch!